projects/common/src/utilities/formatters/duration/display-duration.pipe.ts

```diff
@@ -1,11 +1,19 @@
 import { Pipe, PipeTransform } from '@angular/core';
-import { durationFormatter } from './duration-formatter';
+import { TimeUnit } from './../../../public-api';
+import { TimeDuration, UnitStringType } from './../../../public-api';
+
 
 @Pipe({
   name: 'htDisplayDuration'
 })
 export class DisplayDurationPipe implements PipeTransform {
-  public transform(millis?: number): string {
-    return durationFormatter(millis);
+  public transform(millis?: number, unitStringType: UnitStringType = UnitStringType.Short): string {
+    if(millis === undefined)
+    return '-';
+
+    if(unitStringType === UnitStringType.Short)
+    return new TimeDuration(millis!, TimeUnit.Millisecond).toString();
```

Comment:
this is not the same as the output from durationFormatter. Let's not break existing cases. continue to use `durationFormatter(millis)` for short.

Reply:
reverted